requirements-local.txt

```diff
@@ -1,8 +1,8 @@
 # App packages
-requests==2.25.1
-fcache==0.4.7
-mmh3==2.5.1
-APScheduler==3.7.0
+requests
+fcache
+mmh3
+APScheduler
```

Comment:
what use is this `requirements-local.txt` file anyway?

Reply:
At some point in the distant past, I was using `tox-conda` due to issues with installing mmh3 on OSX.  This was a long time ago and I'm hoping it'll be unnecessary now. 🤞  

(I'll check next time I've got my old dev laptop out.)